mypyc/primitives/bytes_ops.py

```diff
@@ -41,11 +41,38 @@
     error_kind=ERR_MAGIC,
     steals=[True, False])
 
+# bytes[index]
+# bytearray[index]
+method_op(
+    name='__getitem__',
+    arg_types=[bytes_rprimitive, int_rprimitive],
+    return_type=int_rprimitive,
+    c_function_name='CPyBytes_GetItem',
+    error_kind=ERR_MAGIC)
+
+# bytes[index]
+# bytearray[index]
+method_op(
+    name='__getitem__',
+    arg_types=[bytes_rprimitive, short_int_rprimitive],
+    return_type=int_rprimitive,
+    c_function_name='CPyBytes_GetItemShort',
+    error_kind=ERR_MAGIC,
+    priority=2)
+
+# bytearray[index] = int
+method_op(
+    name='__setitem__',
```

Comment:
The `bytes` primitive type doesn't support `__setitem__`. For this to work we'd need a separate `bytearray` primitive type.